### DISCUSSION TITLE
How to train if I have only force data and no energy data?

### DISCUSSION THREAD

**Discussion by utkarshp1161** [mir-group/nequip#354] [Q&A]
How to train if I have only force data and no energy data?

**Comment by Linux-cpp-lisp** [comment]
Hi @utkarshp1161 ,

Just omit `total_energy` from `loss_coeffs`.  Everything else is the same.

**Reply by utkarshp1161** [reply]
you mean, loss_coeffs for total energy needs to be set to zero?

**Reply by Linux-cpp-lisp** [reply]
No, you need to remote `total_energy` from `loss_coeffs` entirely, while leaving `forces` (and whatever other properties you *do* want to train on)

**Reply by Linux-cpp-lisp** [reply]
Setting the coefficient to zero has the same effect, but will still cause an error if there are no `total_energy` labels in your data at all

**Reply by Andy6M** [reply]
Hello @Linux-cpp-lisp,

I came across the previous discussion where @utkarshp1161 inquired about training with only force data and no energy data. You mentioned that they would need to remove total_energy from loss_coeffs entirely while leaving forces.

In my situation, I have the opposite problem: my dataset contains energy data but no force data. Could you please guide me on how to train with only energy data and no force data? Any advice or recommendations would be greatly appreciated.

Thank you for your time and assistance!

**Reply by Linux-cpp-lisp** [reply]
Hi @Andy6M,

https://github.com/mir-group/nequip/blob/main/configs/minimal_eng.yaml should help get you started on that direction.

**Reply by Andy6M** [reply]
@Linux-cpp-lisp Thanks.
